org.eclipse.scout.rt.api/src/main/java/org/eclipse/scout/rt/api/meta/MetaResource.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2010, 2024 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.scout.rt.api.meta;
+
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import javax.security.auth.Subject;
+
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.QueryParam;
+import jakarta.ws.rs.core.Context;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.MultivaluedMap;
+import jakarta.ws.rs.core.Request;
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.core.UriInfo;
+
+import org.eclipse.scout.rt.api.data.meta.MetaVersionInfoDo;
+import org.eclipse.scout.rt.platform.BEANS;
+import org.eclipse.scout.rt.platform.config.CONFIG;
+import org.eclipse.scout.rt.platform.config.PlatformConfigProperties.ApplicationNameProperty;
+import org.eclipse.scout.rt.platform.config.PlatformConfigProperties.ApplicationVersionProperty;
+import org.eclipse.scout.rt.platform.context.RunContext;
+import org.eclipse.scout.rt.platform.nls.NlsLocale;
+import org.eclipse.scout.rt.platform.security.SecurityUtility;
+import org.eclipse.scout.rt.platform.util.CollectionUtility;
+import org.eclipse.scout.rt.platform.util.JarManifestHelper;
+import org.eclipse.scout.rt.platform.util.ObjectUtility;
+import org.eclipse.scout.rt.platform.util.StringUtility;
+import org.eclipse.scout.rt.rest.IRestResource;
+import org.eclipse.scout.rt.rest.doc.ApiDocDescription;
+import org.eclipse.scout.rt.rest.doc.ApiDocGenerator;
+import org.eclipse.scout.rt.rest.doc.ApiDocIgnore;
+
+/**
+ * Provides meta information about the API.
+ */
+@Path("meta")
+@ApiDocDescription(text = "Provides meta information about the API.")
+public class MetaResource implements IRestResource {
+
+  @GET
+  @Path("version")
+  @Produces(MediaType.APPLICATION_JSON)
+  @ApiDocDescription(text = "Returns the API version and application meta data in JSON format.")
+  public MetaVersionInfoDo getMetaVersionInfo() {
+    var apiVersion = BEANS.get(IApiVersion.class);
```

Comment:
This fails in a plain Scout project -> should we provide a basic implementation of IApiVersion also in Scout?